dts/riscv/lowrisc/opentitan_earlgrey.dtsi

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2023 by Rivos Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	#address-cells = < 0x01 >;
+	#size-cells = < 0x01 >;
```

Comment:
please remove redundant spaces `<0x01>` (check others)

Reply:
Done (here and elsewhere).